### ISSUE TITLE
Select individual approvers for the PAF approval

### ISSUE THREAD

**Issue by fourthletter** [HyphaApp/hypha#3198]
### User Story

At the PAF approval stage, OTF wants the project to progress with 1 of 4 approvers approving the PAF and not all the roles approving it. 

### Proposed Solution

1) Can we make this configurable on Hypha such that OTF team can choose who all should review and maybe sequence that as well? So that this configuration is less developer dependent if needed changes in the future.
2) Can we also show in the UI who the PAF is pending on for approval + who all have approved it for further clarity.

**Comment by frjo** [comment]
Closing this in favour of https://github.com/HyphaApp/hypha/issues/3188
